### DISCUSSION TITLE
Support for GTA V - FiveM

### DISCUSSION THREAD

**Discussion by crashzk** [seDirector/App#45] [Ideas]
I would like to know if you intend to put support for the servers of GTA V - FiveM?

https://docs.fivem.net/docs/server-manual/setting-up-a-server-vanilla/#windows

https://docs.fivem.net/docs/server-manual/setting-up-a-server/

**Comment by asheroto** [comment]
I put it under consideration. No plans for it at the moment but will definitely consider it since it's a popular game. :-) Any thoughts about support for Minecraft?

**Comment by crashzk** [comment]
Minecraft I played a lot on console, but I know that to this day many still play it on PC.

I believe that I can offer support for Minecraft, but it would be nice to receive more votes and opinions.
